src/utils/api.ts

```diff
@@ -115,6 +118,25 @@ export class API {
     }
   }
 
+  async getDeployments(
+    id: string,
+  ): Promise<[Deployment[], DeploymentsSummary] | null> {
+    try {
+      return await this.#requestJson(`/projects/${id}/deployments/`);
+    } catch (err) {
+      if (err instanceof APIError && err.code === "projectNotFound") {
+        return null;
+      }
+      throw err;
+    }
+  }
+
+  getLogs(idProject: string, idDeployment: string): AsyncIterable<Logs> {
```

Comment:
```suggestion
  getLogs(projectId: string, deploymentId: string): AsyncIterable<Logs> {
```

This is more consistent with the existing code :)